### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by rcotti-jc**
changed freelancer in User to be an Array, that way a user can claim multiple freelancers
removed claiming from routes and tests, updated using User.freelancer as an array

### CODE REVIEW

File: routes/claim/router.js [hunk 1]
```diff
@@ -56,8 +56,6 @@ router.post('/new', function(req, res) {
         res.status(500).json({ error : 'database error while finding user' }); // TODO: TEST
       } else if (!user) {
         res.status(404).json({ error : 'user not found' });  // TODO: TEST
-      } else if (user.freelancer) {
-        res.status(452).json({ error : 'user id already claiming' }); // TESTED
```
Comment (aronfiechter) [inline routes/claim/router.js:60]: Why is this removed? A single user cannot submit multiple claim requests for the same user, it doesn't make sense.
Is it done client side?

File: routes/freelance/router.js [hunk 2]
```diff
@@ -26,8 +26,7 @@ router.get('/new', function(req, res, next) {
           title: "JobAdvisor - Create Freelancer Profile" ,
           logged: true,
           username: user.username,
-          userFreelancer: user.freelancer,
-          claiming: user.claiming
+          userFreelancer: user.freelancer[0]
```
Comment (aronfiechter) [inline routes/freelance/router.js:29]: Why the first freelancer profile?

File: routes/claim/router.js (hunk 1 above)
Comment (larabr) [reply routes/claim/router.js:60]: What? A user can claim multiple freelancers now, even at the same time

File: routes/freelance/router.js (hunk 2 above)
Comment (larabr) [reply routes/freelance/router.js:29]: Placeholder waiting for the page with all the freelancers for the user

File: routes/claim/router.js (hunk 1 above)
Comment (aronfiechter) [reply routes/claim/router.js:60]: Yeah but he cannot claim _the same_ profile twice, that's nonsense.

File: routes/freelance/router.js (hunk 2 above)
Comment (aronfiechter) [reply routes/freelance/router.js:29]: Kk, we are doing that page on `feature/user-page`. That page will be public, every user can see it, it's just a list of the freelancers owned by that user, each linking to the actual freelancer profile. The links to edit will be on the freelancer profile (if the user is the owner).

File: routes/claim/router.js (hunk 1 above)
Comment (larabr) [reply routes/claim/router.js:60]: Yes but that's checking for multiple claims of different profiles

File: routes/freelance/router.js (hunk 2 above)
Comment (larabr) [reply routes/freelance/router.js:29]: Yes but this is necessary at the moment for example to test emergency

**Review by aronfiechter — CHANGES_REQUESTED** [review]
Something is really wrong in the search router where the key is set, build 549 failed.

**Review by aronfiechter — APPROVED** [review]
